autoload/utils/exec/job.vim

```diff
@@ -39,11 +39,16 @@ function! s:createQuickFix() abort
 endfunction
 
 function! s:vimClose(channel) abort
+    let l:open_qf = get(s:cmake4vim_job, 'open_qf', 0)
```

Comment:
Is this check even needed?

In `s:nVimExit` just `let l:open_qf = s:cmake4vim_job['open_qf']` is used, and this value is always set in `utils#exec#job#run`, so it will always exist.

Reply:
No, I have added this check, because on Windows for some reasons I got sporadic fail that this dictionary doesn't contain this variable.